setup.py

```diff
@@ -64,9 +64,17 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
-    'Invenio[base,auth,metadata,files]{}'.format(invenio_version),
+    'Invenio[base,auth,files]{}'.format(invenio_version),
+    'invenio-indexer>=1.1.1,<1.2.0',
+    'invenio-jsonschemas>=1.0.1,<1.1.0',
+    'invenio-oaiserver>=1.1.1,<1.2.0',
+    'invenio-pidstore>=1.1.0,<1.2.0',
+    'invenio-records-rest>=1.6.4,<1.7.0',
+    'invenio-records-ui>=1.0.1,<1.1.0',
+    'invenio-records>=1.3.0,<1.4.0',
```

Comment:
question: why did we have to explode the metadata bundle again?

Reply:
Because of `invenio-search-ui`! But if we go for moving the `search` ui folder over then this can be imploded again :)

Reply:
Got it.